mkosi

```diff
@@ -1873,6 +1998,7 @@ def print_summary(args):
         sys.stderr.write("        XZ Compression: " + yes_no(args.xz) + "\n")
 
     sys.stderr.write("                Verity: " + yes_no(args.verity) + "\n")
+    sys.stderr.write("              Encryption: " + none_to_na(args.luks_encryption if args.luks_encryption is not None else None) + "\n")
```

Comment:
isn't `none_to_na(args.luks_encryption if args.luks_encryption is not None else None)` identical to `none_to_na(args.luks_encryption)`. /me is now confused!